hal/ubuntu/qemu.sh

```
@@ -0,0 +1,598 @@
+BASE_IMAGE_URL="https://cloud-images.ubuntu.com/noble/current/noble-server-cloudimg-amd64.img"
+BASE_IMAGE="ubuntu-base.qcow2"
+CUSTOM_IMAGE="propeller-cvm.qcow2"
+DISK_SIZE="40G"
+SEED_IMAGE="seed.img"
+USER_DATA="user-data"
+META_DATA="meta-data"
+VM_NAME="propeller-cvm"
+RAM="18192M"
+CPU="4"
+USER="propeller"
+PASSWORD="propeller"
```

Shellcheck (0.11.0)

[warning] PASSWORD appears unused. Verify use (or export if used externally).

(SC2034)